README.md

```diff
@@ -5,27 +5,54 @@ This repository provides a template for creating **feature specifications** for
 
 When a new feature or product is to be developed, a new pull request is created, and contributors are invited to discuss its content and propose suggestions. The goal of a feature specification is to define the expected behavior on a high level and point out corner cases that need to be addressed.
 
+The person in charge of the RP (the owner) is the person assigned to the RP. This allows for the possibility of changing it over time. To change the person assigned to the PR, this should be seen with at least the lest and the new assignee.
```

Comment:
What is `RP`? Is it a typo of `PR` or is it a real acronym? (I ask because the `RP` word is used twice)

Reply:
It's a Grammarly issue I think sorry :/